docker/package_managers/download_pkgs.bzl

```diff
@@ -85,6 +86,8 @@ def _impl(ctx, image_tar = None, packages = None, additional_repos = None, outpu
         output_script: File, overrides ctx.outputs.build_script
         output_metadata: File, overrides ctx.outputs.metadata_csv
     """
+    if types.is_depset(packages):
```

Comment:
Does this check work without `--incompatible_disable_depset_items`?

Reply:
I guess this should work? /cc @alexeagle

Reply:
yes, this only depends on bazel-skylib